server/app/views/applicant/ProgramIndexView.java

```diff
@@ -140,8 +148,7 @@ private DivTag topContent(String titleText, String infoTextLine1, String infoTex
 
   private DivTag mainContent(
       Messages messages,
-      ImmutableList<ProgramDefinition> draftPrograms,
-      ImmutableList<ProgramDefinition> activePrograms,
+      ApplicantService.RelevantPrograms relevantPrograms,
```

Comment:
nit: what do you think about naming this applicationPrograms, or similar.

"relevantPrograms" feels too program focused, which then makes reading relevantPrograms.inProgress harder as the inprogress means "draft application" and not "draft program" which I think is not very clear.

Reply:
Done. Also renamed the type to `ApplicationPrograms`.